examples/with-externals/next.config.js

```diff
@@ -0,0 +1,26 @@
+const webpack = require("webpack")
+
+// Update these to match your package scope name.
+const internalNodeModulesRegExp = /@zeit(?!.*node_modules)/
```

Comment:
A little confused by how this works. The internalNodeModulesRegExp won't replace the scan to our internal node_modules right?

And what's the externalNodeModules? seems like it's used to ignore a re-compile in webpackDevServer?